faq.md

```diff
@@ -0,0 +1,375 @@
+
+8.  **<a name="remove_dir">My backup set contains some files that I just realized I don't want/need backed up. How do I remove them from the backup volume to save space?</a>**
+
+    The only official way to remove files from an rdiff-backup repository is by letting them expire using the --remove-older-than option. Deleting increments from the rdiff-backup-data directory will prevent you from recovering those files, but shouldn't prevent the rest of the repository from being restored.
+
+9.  **<a name="solaris">Does rdiff-backup work under Solaris?</a>**
+
+    There may be a problem with rdiff-backup and Solaris' libthread. Adding "ulimit -n unlimited" may fix the problem though. Here is a post by Kevin Spicer on the subject:
+
+    <pre>Subject: RE: Crash report....still not^H^H^H working
+    From: "Spicer, Kevin" <kevin.spicer@bmrb.co.uk>
+    Date: Sat, 11 May 2002 23:36:42 +0100
+    To: rdiff-backup@keywest.Stanford.EDU
+
+    Quick mail to follow up on this..
+    My rdiff backup (on Solaris 2.6 if you remember) has now worked
+    reliably for nearly two weeks after I added...
+
+        ulimit -n unlimited
+
+    to the start of my cron job and created a wrapper script on the remote
+    machine which looked like this...
+
+        ulimit -n unlimited
+        rdiff-backup --server
+        exit
+
+    And changed the remote schema on the command line of rdiff-backup to
+    call the wrapper script rather than rdiff-backup itself on the remote
+    machine.  As for the /dev/zero thing I've done a bit of Googleing and
+    it seems that /dev/zero is used internally by libthread on Solaris
+    (which doesn't really explain why its opening more than 64 files - but
+    at least I think I've now got round it).
+    </pre>
+
+10.  **<a name="speed">How fast is rdiff-backup? Can it be run on large data sets?</a>**
+
+    rdiff-backup can be limited by the CPU, disk IO, or available bandwidth, and the length of a session can be affected by the amount of data, how much the data changed, and how many files are present. That said, in the typical case the number/size of changed files is relatively small compared to that of unchanged files, and rdiff-backup is often either CPU or bandwidth bound, and takes time proportional to the total number of files. Initial mirrorings will usually be bandwidth or disk bound, and will take much longer than subsequent updates.
```

Comment:
The indent you've got here is causing markdown to render the answers as a `<pre>`, which isn't really suitable.